Remotion/Data/DomainObjects.UnitTests/IntegrationTests/Synchronization/SyncStateTest.cs

```diff
@@ -52,7 +52,11 @@ public void CollectionItems_Unsynchronized_WithUnload ()
       var order = DomainObjectIDs.Order1.GetObject<Order>();
       order.OrderItems.EnsureDataComplete();
 
-      var orderItemID = RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction<OrderItem, Order>(order.ID, (oi, o) => oi.Order = o);
```

Comment:
marked file (please move property initialization up to object load/initialization)

Reply:
Same problem as above: I cannot set `OrderItem.Product` before creating the `OrderItem`, which is done in `RelationInconcsistenciesTestHelper.CreateObjectAndSetRelationInOtherTransaction()`.